### ISSUE TITLE
Handle failure to lock vmdk during container start

### ISSUE THREAD

**Issue by hickeng** [vmware/vic#5571]
**VIC version:**
dev - 732f8779ba924a582464b5c6c9317da0bae7f445

**Deployment details:**
Generic VCH with default volume store configured

```
vic-machine-linux create --target=https://192.168.78.128 --image-store=datastore1/localtls/images --compute-resource= --name=localtls --container-network=VMExternal:external --volume-store=datastore1/volumes:default --volume-store=datastore1/other-volumes:others --public-network-ip=192.168.78.127/24 --public-network-gateway=192.168.78.2 --dns-server=10.132.71.1 --dns-server=10.132.71.2 --thumbprint= --debug=2
```

**Steps to reproduce:**
```
$ docker volume create
f6cbafbf-0b8e-4368-af3d-035e0c409217
$ docker run -dit -v f6cbafbf-0b8e-4368-af3d-035e0c409217:/var/tmp busybox
Unable to find image 'busybox:latest' locally
latest: Pulling from library/busybox
27144aa8f1b9: Pull complete
a3ed95caeb02: Pull complete
Digest: sha256:be3c11fdba7cfe299214e46edc642e09514dbb9bbefcd0d3836c05a1e0cd0642
Status: Downloaded newer image for library/busybox:latest
eeea17a4de4ff93410c885dba2ba6073daa2cb5fc4d06d4aa50e9b55c13c0922
$ docker run -dit -v f6cbafbf-0b8e-4368-af3d-035e0c409217:/var/tmp busybox
8f534a02bffa167b0fb229b7a9659d6ebc3f246d50af335387f4d45d638a241a
docker: Error response from daemon: Server error from portlayer: File system specific implementation of LookupAndOpen[file] failed.
$ docker ps -a
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS                     PORTS               NAMES
8f534a02bffa        busybox             "sh"                5 hours ago         Starting                                       wonderful_goldberg
eeea17a4de4f        busybox             "sh"                5 hours ago         Exited (143) 5 hours ago                       clever_goldwasser
```

**Actual behavior:**
ContainerVM fails to start
docker ps lists container as `Starting` indefinitely

**Expected behavior:**
ContainerVM fails to start
docker ps lists container as `Stopped` to reflect it's power state (off)

**Comment by hickeng** [comment]
This locking issue is going to crop up more frequently with the introduction of `docker cp` support, so marking fixing this failure path handling as a part of the feature work.

**Comment by hickeng** [comment]
@matthewavery @jzt @sflxn This is one example of a path that hits an in-use vmdk - in this specific case we need to return a much more usable error to the user.
In the implementation of Import/Export we'll need to use this to fail-over to the live paths.

**Comment by corrieb** [comment]
Assuming that since this is part of the Docker cp epic, it will be addressed for 1.2.

**Comment by emlin** [comment]
The PR test passed, just ui build still fail
